.github/workflows/build.yaml

```
@@ -146,3 +145,37 @@ jobs:
           OPENSSL_ROOT_DIR: "/usr/lib/aarch64-linux-gnu"
           OPENSSL_STATIC: "1"
           SASL2_STATIC: "0"
+        # Add the fix as a pre-run script right before cargo build runs
+        # This is a workaround since actions-rs/cargo doesn't support pre-commands
+        run: |
```

actionlint (1.7.4)

this step is for running action since it contains at least one of "uses", "with" keys, but also contains "run" key which is used for running shell command

(syntax-check)